apps/worker/src/app/workflow/usecases/add-job/add-job.usecase.ts

```diff
@@ -157,9 +161,41 @@ export class AddJob {
 
     const delay = this.getExecutionDelayAmount(filtered, digestAmount, delayAmount);
 
+    await this.validateDeferDuration(delay, job, command);
+
     await this.queueJob(job, delay);
   }
 
+  private async validateDeferDuration(delay: number, job: JobEntity, command: AddJobCommand) {
+    const issues = await this.validateControlByTierUsecase.execute(
+      ValidateControlByTierCommand.create({
+        deferDuration: delay,
+        stepType: job.type,
+        organizationId: command.organizationId,
+      })
+    );
+
+    if (Object.keys(issues).length > 0) {
+      const tierIssues = (issues as Record<string, ContentIssue[]>).tier;
+      if (tierIssues) {
+        Logger.warn({ issues: tierIssues, jobId: job._id }, 'Defer duration limit exceeded for job', LOG_CONTEXT);
```

Comment:
What does `Defer` mean in the context of a maximum duration limit? The naming is weird to me. I would just go with:
```suggestion
        Logger.warn({ issues: tierIssues, jobId: job._id }, 'Maximum duration limit exceeded for job', LOG_CONTEXT);
```

Reply:
I think we should stick with this term since it's already used in the 'add job' use case. If needed, I can ensure it's applied consistently across the project, and i will raise it with the team.

We need a clear way to define two things in the project:  
1. The time we delay or notification, without explicitly using 'delay' or 'digest.'  
2. A term for actions that aren’t executed immediately (deferred actions).  

Having consistent terms is important because the current lack of clarity causes confusion in the code.  

I'm open to using a different term as long as we use it consistently and everyone understands it.